budget-binder-multiplatform-app/src/commonMain/kotlin/de/hsfl/budgetBinder/presentation/viewmodel/dashboard/DashboardViewModel.kt

```diff
@@ -113,21 +119,26 @@ class DashboardViewModel(
     }
 
 
-    private fun getEntriesByCategory(id: Int? = null, period: String? = null, onSuccess: (List<Entry>) -> Unit) =
+    private fun getEntriesByCategory(
+        id: Int? = null,
+        period: String? = null,
+        onSuccess: (List<Entry>) -> Unit
+    ) =
         scope.launch {
```

Comment:
scope.launch einfach in die Zeile darüber neben das gleich packen

Reply:
Ey jetzt fang nicht an Cedriks "fehler" in meiner Request anzumerken, nur weil ich formatiert habe! 😂 wenn ich schon dabei bin update ich generell aber mal das DashboardViewModel